pkg/components/metadata.go

```diff
@@ -24,5 +24,6 @@ type Metadata struct {
 
 // HelmMetadata stores Helm-related information about a component that is needed when managing component using Helm.
 type HelmMetadata struct {
-	Wait bool
+	Wait            bool
+	NamespaceConfig string
```

Comment:
IMO the approach for namespaces implemented in #646 is better and we should use that instead of patches introduced here.

Reply:
changed according to current state of master.